### ISSUE TITLE
add separate people's page ranked by their reputation=)

### ISSUE THREAD

**Issue by Erol314** [brendt/rfc-vote#215]
why?:
1. just to search more easily some fellas, 
    (in case you want to read more of their arguments/comments but don't want to re-read all arguments under all rfcs to pinpoint their comment )
2. looking on the sorted ranks table may stimulate people to more activity (like in a game)

**Comment by brendt** [comment]
Yeah, I think this makes sense. It was always the goal of reputation to be something visible, to encourage people to participate. So I really like this idea 👌 Not sure how we'd design the page, and where we should put it in the menu though. Feedback appreciated!

**Comment by Erol314** [comment]
The idea was inspired from Codewars, so the ranking like this is fine, I think
![image](https://github.com/brendt/rfc-vote/assets/24621490/c2f3b7c9-2cd6-4667-81da-7daf487b33ce)

- Search by name/nick


1. User Position
2. User
    1. you said that user will unlock more privileges with more activity, this is the privilege name(like member,  veteran ...)
    2. Avatar
    3. nick
3. User Badges
4. Total reputation



Putting in the menu under "Community" is fine,

**Comment by brendt** [comment]
Good start! Do you want to give it a shot?

**Comment by Plytas** [comment]
Maybe this could be combined with #175?

**Comment by brendt** [comment]
Well that would be a public facing page, so it would have different styling

**Comment by Erol314** [comment]
> Good start! Do you want to give it a shot?

I Id like to do it=)

**Comment by brendt** [comment]
Go for it!
